src/Cell.tsx

```diff
@@ -107,6 +107,7 @@ function Cell<R, SR>({
       aria-colindex={column.idx + 1} // aria-colindex is 1-based
       aria-selected={isCellSelected}
       aria-colspan={colSpan}
+      aria-readonly={isCellEditable(column, row) ? true : undefined}
```

Comment:
No?
```suggestion
      aria-readonly={isCellEditable(column, row)}
```

Reply:
IIRC, it is usually recommended to skip setting `false` value. I check a few grids and found 1 that uses `aria-readonly` and it does not set it when value is editable
https://www.grapecity.com/wijmo/demos/Grid/Editing/Read-only/purejs

Reply:
🤔 
```suggestion
      aria-readonly={isCellEditable(column, row) || undefined}
```